sdk/identity/identity/samples/AzureCacheForRedis/ioredis.md

```diff
@@ -0,0 +1,273 @@
+## Azure Cache for Redis: Azure AD with ioredis client library
+
+### Table of contents
+
+- [Prerequisites](#prerequisites)
+- [Authenticate with Azure AD - Hello World](#authenticate-with-azure-ad-ioredis-hello-world)
```

Comment:
I still find "Hello World" offputting.

What I don't want is for someone to search "@azure/identity hello world" and end up on a detailed sample about how to auth a hosted redis client. I don't think there's anything wrong with just calling this sample "Authenticate a client" rather than "hello world." The latter is just not informative of what the content of the sample actually seeks to accomplish.

CC @scottaddie

In general, I think if we find ourselves reaching for "it's the hello world of `X`" as a way to explain the sample, then the sample should just be called `X` instead. A less problematic version of this is "getting started," but in this case it's just about authenticating the client, so why not just call it that?

Reply:
@KarishmaGhiya Will makes a great point here. It's something I overlooked. Let's change the heading to "Authenticate with Azure AD - Authenticate a client".